### ISSUE TITLE
Import addons/sale/i18n/ja.po got error

### ISSUE THREAD

**Issue by telesoho** [odoo/odoo#33458]
Impacted versions:
odoo 12

Steps to reproduce:
1. From menu select "Settings\Translations\Load a Translation"
2. Select Japanese Language
3. Check "Overwrite Existing" 
4. Press load button.

Current behavior:

Throw the following exeption:
```log
2019-05-17 07:29:32,232 32129 INFO test odoo.tools.translate: loading /home/telesoho/prjs/odoo/community/addons/sale/i18n/ja.po 
2019-05-17 07:29:32,288 32129 ERROR test odoo.sql_db: bad query: b" INSERT INTO ir_translation(name, lang, res_id, src, type, value, module, state, comments)\n                           SELECT name, lang, res_id, src, type, value, module, state, comments\n                           FROM tmp_ir_translation_import\n                           WHERE type = 'model'\n                           AND noupdate IS NOT TRUE\n                           ON CONFLICT (type, lang, name, res_id) WHERE type = 'model'\n                            DO UPDATE SET (name, lang, res_id, src, type, value, module, state, comments) = (EXCLUDED.name, EXCLUDED.lang, EXCLUDED.res_id, EXCLUDED.src, EXCLUDED.type, EXCLUDED.value, EXCLUDED.module, EXCLUDED.state, EXCLUDED.comments)\n                            WHERE EXCLUDED.value IS NOT NULL AND EXCLUDED.value != '';\n                       "
ERROR: ON CONFLICT DO UPDATE command cannot affect row a second time
HINT:  Ensure that no rows proposed for insertion within the same command have duplicate constrained values.
 
2019-05-17 07:29:32,289 32129 ERROR test odoo.http: Exception during JSON request handling. 
Traceback (most recent call last):
  File "/home/telesoho/prjs/odoo/community/odoo/http.py", line 654, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/home/telesoho/prjs/odoo/community/odoo/http.py", line 312, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/home/telesoho/prjs/odoo/community/odoo/tools/pycompat.py", line 87, in reraise
    raise value
  File "/home/telesoho/prjs/odoo/community/odoo/http.py", line 696, in dispatch
    result = self._call_function(**self.params)
  File "/home/telesoho/prjs/odoo/community/odoo/http.py", line 344, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/home/telesoho/prjs/odoo/community/odoo/service/model.py", line 97, in wrapper
    return f(dbname, *args, **kwargs)
  File "/home/telesoho/prjs/odoo/community/odoo/http.py", line 337, in checked_call
    result = self.endpoint(*a, **kw)
  File "/home/telesoho/prjs/odoo/community/odoo/http.py", line 939, in __call__
    return self.method(*args, **kw)
  File "/home/telesoho/prjs/odoo/community/odoo/http.py", line 517, in response_wrap
    response = f(*args, **kw)
  File "/home/telesoho/prjs/odoo/community/addons/web/controllers/main.py", line 966, in call_button
    action = self._call_kw(model, method, args, {})
  File "/home/telesoho/prjs/odoo/community/addons/web/controllers/main.py", line 954, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/home/telesoho/prjs/odoo/community/odoo/api.py", line 749, in call_kw
    return _call_kw_multi(method, model, args, kwargs)
  File "/home/telesoho/prjs/odoo/community/odoo/api.py", line 736, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/home/telesoho/prjs/odoo/community/addons/website/wizard/base_language_install.py", line 25, in lang_install
    action = super(BaseLanguageInstall, self).lang_install()
  File "/home/telesoho/prjs/odoo/community/odoo/addons/base/wizard/base_language_install.py", line 36, in lang_install
    mods.with_context(overwrite=self.overwrite)._update_translations(self.lang)
  File "/home/telesoho/prjs/odoo/community/odoo/addons/base/models/ir_module.py", line 885, in _update_translations
    self.env['ir.translation'].load_module_terms(mod_names, filter_lang)
  File "/home/telesoho/prjs/odoo/community/odoo/addons/base/models/ir_translation.py", line 805, in load_module_terms
    tools.trans_load(self._cr, base_trans_file, lang, verbose=False, module_name=module_name, context=context)
  File "/home/telesoho/prjs/odoo/community/odoo/tools/translate.py", line 1013, in trans_load
    result = trans_load_data(cr, fileobj, fileformat, lang, verbose=verbose, module_name=module_name, context=context)
  File "/home/telesoho/prjs/odoo/community/odoo/tools/translate.py", line 1157, in trans_load_data
    irt_cursor.finish()
  File "/home/telesoho/prjs/odoo/community/odoo/addons/base/models/ir_translation.py", line 156, in finish
    """ % (self._model_table, self._table))
  File "/home/telesoho/prjs/odoo/community/odoo/sql_db.py", line 148, in wrapper
    return f(self, *args, **kwargs)
  File "/home/telesoho/prjs/odoo/community/odoo/sql_db.py", line 225, in execute
    res = self._obj.execute(query, params)
psycopg2.ProgrammingError: ON CONFLICT DO UPDATE command cannot affect row a second time
HINT:  Ensure that no rows proposed for insertion within the same command have duplicate constrained values.
```

When I remove "addons/sale/i18n/ja.po" file, all things ok.

**Comment by mart-e** [comment]
Hello,

Is your code up to date?
I believe we have fixed similar issue not long ago at 6afec6b6b8d2a95264f1a775f8dcee9a90d3fda7
Can you reproduce it on a 12.0 [runbot](http://runbot.odoo.com/) ?

**Comment by telesoho** [comment]
It's work, thank you!
